test/Azure.Functions.Cli.Tests/E2E/StartTests.cs

```diff
@@ -294,7 +295,8 @@ await CliTester.Run(new RunConfiguration[]
                    {
                        "init . --worker-runtime node",
                        "new --template \"Http trigger\" --name HttpTrigger",
-                   }
+                   },
+                   CommandTimeout = TimeSpan.FromSeconds(300)
```

Comment:
Why is this timeout needed on these tests?

Reply:
@liliankasem Those testcases are failing, whenever we create a functionapp with -node runtime. npm will take some time to install all the packages required. it needs some time to install when using -init then we can create template of the trigger.